### PR TITLE
add a test for http redirects

### PR DESCRIPTION
**Pull request by pb82**
## Motivation

Adds a test for the redirect handling in the http service.

### CODE REVIEW

**Comment by pb82** [comment]
@secondsun i changed to test to use mockwebserver but i need to get this published first: https://github.com/aerogear/aerogear-parent/pull/82

File: core/src/test/java/org/aerogear/mobile/core/http/OkHttpServiceModuleTest.java [hunk 1]
```diff
@@ -94,4 +96,36 @@ public void testCompleteHandlerCalledInErrorCase() {
...
+    public void testRedirectsShouldBeSuccessful() {
+        OkHttpClient client = new OkHttpClient.Builder()
+            .followRedirects(false)
+            .followSslRedirects(false)
+            .build();
+
+        HttpServiceModule module = new OkHttpServiceModule(client);
+
+        HttpRequest request = module.newRequest();
+        request.get("https://jigsaw.w3.org/HTTP/300/302.html");
```
Comment (danielpassos) [inline core/src/test/java/org/aerogear/mobile/core/http/OkHttpServiceModuleTest.java]: I see you are going to replace it with the mock server, just wanna leave a tip here. When I need to test some HTTP status I always use https://httpstat.us/

**Comment by secondsun** [comment]
@pb82 The parent should be published now (can @danielpassos  confirm).  I've gone ahead and flipped my bit to "Accepted" since you said you were going to add the mock server when it is available.

**Comment by danielpassos** [comment]
http://search.maven.org/#artifactdetails%7Corg.jboss.aerogear%7Caerogear-parent%7C1.1.11%7Cpom

**Comment by pb82** [comment]
@danielpassos @secondsun thanks, i'll push the changes to mockserver later today

**Pull request merged**